### ISSUE TITLE
does not seem to be node 6 compatible?

### ISSUE THREAD

**Issue by makc** [stackgl/headless-gl#59]
after updating to node 6 I get this:
```
> gl@3.0.3 install /Users/makc/Desktop/xxxxxxx/3d/node_modules/gl
> prebuild --install

prebuild WARN install No prebuilt binaries found (target=v6.0.0 arch=x64 platform=darwin)
gyp: angle/build/standalone.gypi not found (cwd: /Users/makc/Desktop/xxxxxxx/3d/node_modules/gl) while reading includes of angle/src/angle.gyp while loading dependencies of binding.gyp while trying to load binding.gyp
prebuild ERR! configure error 
prebuild ERR! stack Error: `gyp` failed with exit code: 1
prebuild ERR! stack     at ChildProcess.onCpExit (/Users/makc/Desktop/xxxxxxx/3d/node_modules/node-gyp/lib/configure.js:305:16)
prebuild ERR! stack     at emitTwo (events.js:106:13)
prebuild ERR! stack     at ChildProcess.emit (events.js:191:7)
prebuild ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:204:12)
prebuild ERR! not ok 
```
is this even fixable on my end?

**Comment by dfcreative** [comment]
I always had this issue, so just clone via git

**Comment by mikolalysenko** [comment]
This seems to be a problem with the way prebuild is configured.  @ralphtheninja, sorry to bug you, but do you know what is going on?

**Comment by makc** [comment]
5.11.0 is fine, so I just downgraded, but long-term solution would be nice

**Comment by mikolalysenko** [comment]
Ok, I sorted it out.  This all should be fixed in 3.0.5
